workshops/FCR/README.md

```diff
@@ -0,0 +1,44 @@
+
+# Starter Pack
+Just go to [teachablemachine.withgoogle.com](https://teachablemachine.withgoogle.com)and click Get Started. No need to make an account or log in. [This playlist of tutorial videos](https://www.youtube.com/playlist?list=PLJfHZtseuscuTQfodmFnbZ3rBgCWsRT9t) are built into the tool to help you along the way. You can also check out [Dan Shiffman’s Coding Train video](https://www.youtube.com/watch?v=kwcillcWOg0&list=PLRqwX-V7Uu6aJwX0rFP-7ccA6ivsPDsK5&index=2&t=0s) on Teachable Machine. Three starter projects for recognizing [fruit](https://medium.com/@warronbebster/teachable-machine-tutorial-bananameter-4bfffa765866), [head tilts](https://medium.com/@warronbebster/teachable-machine-tutorial-head-tilt-f4f6116f491), or [sounds](https://medium.com/@warronbebster/teachable-machine-tutorial-snap-clap-whistle-4212fd7f3555) you make.You can currently train Teachable Machine with images (pulled from your webcam or image files), sounds (in one-second snippets from your mic), and poses (where the computer guesses the position of your arms, legs, etc from an image). More types of training may be coming soon :)
+
+# Getting start with Facial Expression Recognition
+Facial expressions convey the emotional state of an individual to observer what is at the back of mind. What we do while we speak often says more than the actual words. There are six basic types of emotions
+__1.Happiness 2.Sadness 3.Fear 4.Anger 5.Suprise 6.Disgust.__<br />
+![](https://cloud-kmaqeh4qc.vercel.app/happy1.gif)
+![](https://cloud-kmaqeh4qc.vercel.app/sad1.gif) 
+![](https://cloud-kmaqeh4qc.vercel.app/fear1.gif) <br />
+
+![](https://cloud-kmaqeh4qc.vercel.app/angry1.gif)
+![](https://cloud-kmaqeh4qc.vercel.app/suprise1.gif) 
+![](https://cloud-kmaqeh4qc.vercel.app/disgust1.gif)
+
+## How facial expression recognition works?
+ A convolutional neural network for classifying human emotions from dynamic facial expressions in real time. We use transfer learning on the fully connected layers of an existing convolutional neural network which was pretrained for human emotion classification. A variety of datasets, as well as our own unique
+image dataset, will be use to train the model. Finally, a live video stream connected to a
+face detector feeds images to the neural network. The network subsequently classifies an arbitrary number of faces
+per image simultaneously in real time and display the probability of emotion in range of 0 to 1. Facial expression recognition work in 2 parts:
+1. Facial Detection:- The ability to detect the location of face in any input image or frame. The output is bounding box of the deteced faces.<br />
+2. Emotion Detection:- Classifying the emotion on the face as happy, angry, sad, surprise, disgust or fear.<br />
```

Comment:
A convolutional neural network for classifying human emotions from dynamic facial expressions in real-time. We use transfer learning on the fully connected layers of an existing convolutional neural network which was trained for human emotion classification. A variety of datasets, as well as our own unique
		image dataset will be used to train the model. Finally, a live video stream connected to a
		A face detector feeds images to the neural network. The network subsequently classifies an arbitrary number of faces
		per image simultaneously in real-time and display the probability of emotion in the range of 0 to 1. Facial expression recognition work in 2 parts:
		1. Facial Detection:- The ability to detect the location of the face in any input image or frame. The output is a bounding box of the detected faces.<br />
		2. Emotion Detection:- Classifying the emotion on the face as happy, angry, sad, surprise, disgust, or fear.<br />

Reply:
Just corrected some grammer